webviews/preview_compiled/components/SchemaTab.tsx

```
@@ -0,0 +1,78 @@
+      header: "Name",
+    },
+    {
+      accessorKey: "type",
+      header: "Type",
+    },
+    {
+      accessorKey: "description",
+      header: "Description",
+      cell: ({ getValue, row }) => {
+          const initialValue = getValue() as string || "";
+          const [value, setValue] = React.useState(initialValue);
```

Biome (2.3.14)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)